mmaction/models/utils/graph.py

```diff
@@ -3,6 +3,7 @@
 
 
 def get_hop_distance(num_node, edge, max_hop=1):
+    """Get n-hop distance matrix by edges."""
```

Comment:
Please add details descriptions like arguments and return values for utility functions.